content/billing/billing.md

```diff
@@ -51,6 +51,10 @@ Invoices that have not been paid on the due date are shown as "Overdue" or "Fail
 
 As soon as there is a failed invoice payment, your subscription will be put on hold and running builds will be disabled. In order to continue building all invoices should be paid.
 
-Team or account owners can manually retry the payment by clicking on the overdue invoice in **Billing history** and selecting **Pay invoice** on the popup with invoice details.
+Team or account owners can manually retry the payment by clicking on the overdue/pending invoice in **Billing history** and clicking the blue **$** icon in the **Invoice** column. It will redirect you to the Stripe-based payment website where you are required to add your credit card details and click **Make Payment**.
+
+{{<notebox>}}
+**Pending invoices**: You may experience failed payments, due to the disabled international payments on your credit cards or automatic payments not allowed in which case the first payment has to be made manually, to comply with new RBI regulations for Indian cardholders.
```

Comment:
```suggestion
**Pending invoices**: Some invoices may remain pending due to issues with the payment. This can happen when international payments on your card are disabled or when automatic payments are not allowed in which case the first payment has to be made manually to comply with the new RBI regulations for Indian cardholders. Team or account owners can then manually retry the payment as described above.
```